filebeat/module/elasticsearch/audit/test/test-audit.log-expected.json

```diff
@@ -0,0 +1,133 @@
+[
+    {
+        "@timestamp": "2018-10-31T09:34:25.109Z",
+        "ecs.version": "1.0.0-beta2",
+        "elasticsearch.audit.event_type": "authentication_failed",
+        "elasticsearch.audit.layer": "rest",
+        "elasticsearch.audit.origin_address": "::1",
+        "elasticsearch.audit.origin_port": 61598,
+        "elasticsearch.audit.origin_type": "rest",
+        "elasticsearch.audit.principal": "elastic",
+        "elasticsearch.audit.uri": "/_xpack/security/user/beats_system/_password",
+        "elasticsearch.node.id": "DSiWcTyeThWtUXLB9J0BMw",
+        "event.dataset": "audit",
+        "event.module": "elasticsearch",
+        "input.type": "log",
+        "log.offset": 0,
+        "message": "{\"@timestamp\":\"2018-10-31T09:34:25,109\", \"node.id\":\"DSiWcTyeThWtUXLB9J0BMw\", \"event.type\":\"rest\", \"event.action\":\"authentication_failed\", \"user.name\":\"elastic\", \"origin.type\":\"rest\", \"origin.address\":\"[::1]:61598\", \"url.path\":\"/_xpack/security/user/beats_system/_password\"}",
+        "service.name": "elasticsearch"
+    },
+    {
+        "@timestamp": "2018-10-31T09:34:25.207Z",
+        "ecs.version": "1.0.0-beta2",
+        "elasticsearch.audit.event_type": "authentication_failed",
+        "elasticsearch.audit.layer": "rest",
+        "elasticsearch.audit.origin_address": "::1",
+        "elasticsearch.audit.origin_port": 61599,
+        "elasticsearch.audit.origin_type": "rest",
+        "elasticsearch.audit.principal": "elastic",
+        "elasticsearch.audit.uri": "/_xpack/security/user/remote_monitoring_user/_password",
+        "elasticsearch.node.id": "DSiWcTyeThWtUXLB9J0BMw",
+        "event.dataset": "audit",
+        "event.module": "elasticsearch",
+        "input.type": "log",
+        "log.offset": 274,
+        "message": "{\"@timestamp\":\"2018-10-31T09:34:25,207\", \"node.id\":\"DSiWcTyeThWtUXLB9J0BMw\", \"event.type\":\"rest\", \"event.action\":\"authentication_failed\", \"user.name\":\"elastic\", \"origin.type\":\"rest\", \"origin.address\":\"[::1]:61599\", \"url.path\":\"/_xpack/security/user/remote_monitoring_user/_password\"}",
+        "service.name": "elasticsearch"
+    },
+    {
+        "@timestamp": "2018-10-31T09:35:11.428Z",
+        "ecs.version": "1.0.0-beta2",
+        "elasticsearch.audit.action": "cluster:admin/xpack/security/realm/cache/clear",
+        "elasticsearch.audit.event_type": "access_granted",
+        "elasticsearch.audit.layer": "transport",
+        "elasticsearch.audit.origin_address": "127.0.0.1",
```

Comment:
Looks like quite a few fields here we should map to ECS (follow up PR).

Reply:
Agreed. I've never done an ECS conversion before. Would you mind pointing me to a PR that did a similar conversion and I could use as a reference? Thanks!

Reply:
Here you have quite a list of PR's: https://github.com/elastic/beats/issues/8655